src/main/docs/guide/whatsnew.adoc

```diff
@@ -1 +1 @@
-Since Micronaut MicroStream 2, you can use <<storageTargets, Storage Targets>> such as PostgreSQL, Amazon S3 and DynamoDB.
+Since Micronaut EclipseStore 2, you can use <<storageTargets, Storage Targets>> such as PostgreSQL, Amazon S3 and DynamoDB.
```

Comment:
This doesn't seem right. There is no EclipseStore 2. I think the line can be deleted.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-eclipsestore/pull/1/commits/2ee181a999c0b7fa8cda54f87af54f9ece8c369a